Settings-default.toml

```diff
@@ -175,3 +175,6 @@ endpoint = "127.0.0.1:6831"
 
 [data]
 volumes = { test_data = "test_data/"}
+
```

Comment:
The coverage is quite low. Is there a blind spot in this impl?

Reply:
Actually, the test didn't use the cache properly, because it wasn't filled yet. It's filled asynchronously. The test coverage should be better now: ae20835ba3f213a20e99ad74c583452a0aaf2089